setup.py

```diff
@@ -12,7 +12,7 @@
 
 setup(
     name='pulp-ansible',
-    version='0.2.0b1',
+    version='0.2.0b1.dev',
```

Comment:
Also every plugin has the version in a second place too. It's the __version__ = in the __init__.py file of the package. That needs to be changed for any of them also.

Reply:
Is it for towncrier purposes?

Reply:
Not especially for towncrier, although towncrier does use it. It's from pep396 https://www.python.org/dev/peps/pep-0396/. If we were doing it super to the spec we would derive the setup.py from __version__ or vice versa as in [step 8](https://www.python.org/dev/peps/pep-0396/#specification). I think using `bumpversion` would be good too.